content/en/docs/Getting Started/pipelines.md

```diff
@@ -1,191 +1,165 @@
 ---
 title: "Getting Started with Pipelines"
 linkTitle: "Getting Started with Pipelines"
-weight: 1
+weight: 2
 description: >
-  Prerequisites, Installation, and Basic Usage
+  Create and run your first Tekton pipeline
+
 ---
 
-## Prerequisites
+This tutorial shows you how to:
 
-* Completed the [Getting Started example](/docs/getting-started/).
-
-## Extending your first CI/CD Workflow with a second Task and a Pipeline
-
-As you learned previously, with Tekton, each operation in your CI/CD workflow becomes a `Step`,
-which is executed with a container image you specify. `Steps` are then
-organized in `Tasks`, which run as a [Kubernetes pod](https://kubernetes.io/docs/concepts/workloads/pods/)
-in your cluster. You can further organize `Tasks` into `Pipelines`, which 
-can control the order of execution of several `Tasks`. 
-
-To create a second `Task`, create a Kubernetes object using the Tekton API with
-the kind `Task`. The following YAML file specifies a `Task` with one simple
-`Step`, which prints a `Goodbye World!` message using
-[the official Ubuntu image](https://hub.docker.com/_/ubuntu/):
-
-```yaml
-apiVersion: tekton.dev/v1beta1
-kind: Task
-metadata:
-  name: goodbye
-spec:
-  steps:
-    - name: goodbye
-      image: ubuntu
-      script: |
-        #!/bin/bash
-        echo "Goodbye World!"
-```
+- Create two tasks.
+- Create a pipeline containing your tasks.
+- Use `PipelineRun` to instantiate and run the pipeline containing your tasks.
 
-Write the YAML above to a file named `task-goodbye.yaml`, and apply it to your Kubernetes cluster:
+For this tutorial we are going to use [minikube][minikube] to run the commands
+locally.
 
-```bash
-kubectl apply -f task-goodbye.yaml
-```
+## Prerequisites
 
-To run this task with Tekton, you need to create a `TaskRun`, which is
-another Kubernetes object used to specify run time information for a `Task`. 
+- Complete the [Getting started with tasks](/docs/getting-started/tasks/)
+	tutorial. *Do not clean up your resources*, skip the last section.
+
+- [Install the Tekton CLI](/docs/cli/).
 
-To view this `TaskRun` object you can run the following Tekton CLI (`tkn`) command:
+## Creating and running a second task
 
-```shell
-tkn task start goodbye --dry-run
-```
+You already have a *Hello World!* task. To create a second *Goodbye World!*
+task:
 
-After running the command above, the following `TaskRun` definition should be shown:
+1.  Create a new file named  `goodbye-world.yaml` and add the following
+    content:
 
-```yaml
-apiVersion: tekton.dev/v1beta1
-kind: TaskRun
-metadata:
-  generateName: goodbye-run-
-spec:
-  taskRef:
-    name: goodbye
-```
+    ```yaml
+	  apiVersion: tekton.dev/v1beta1
+	  kind: Task
+	  metadata:
+	     name: goodbye
+	  spec:
+	     steps:
+	        - name: goodbye
+	          image: ubuntu
+	          script: |
+	             #!/usr/bin/env bash
+	             echo "Goodbye World!"
```

Comment:
there's some extra whitespace at the beginning of the yaml block that shows up when i try to copy paste from this block:

![image](https://user-images.githubusercontent.com/432502/154171090-7b508ac6-fc8e-43ba-854e-001994db31da.png)